src/app/containers/ATIAnalytics/atiUrl.js

```diff
@@ -12,7 +12,12 @@ import {
   sanitise,
 } from '../../lib/analyticsUtils';
 
-export const atiBaseUrl = 'https://a1.api.bbc.co.uk/hit.xiti?';
+export const atiBaseUrl = env => {
+  if (env === 'local' || env === 'test') {
+    return 'https://logws1363.ati-host.net?';
+  }
+  return 'https://a1.api.bbc.co.uk/hit.xiti?';
```

Comment:
does it make more sense to store these values in envConfig? wouldn't need if/else logic then either, right?
e.g
```suggestion
  return process.env.SIMORGH_ATI_BASEURL;
```

Reply:
Based on this do we need the if statement? Surely we can have the local and test config have `SIMORGH_ATI_BASEURL` switch between `https://logws1363.ati-host.net?` and `https://a1.api.bbc.co.uk/hit.xiti?` based on the env config?